_episodes/markdown.md

```diff
@@ -266,10 +259,13 @@ You can define either
 > ~~~
 > {: .source }
 >
+<<<<<<< HEAD
+=======
 > to create the following rendered view, where both links point to <https://carpentries.org/>.
 >
 > ![Preview tab of inline and reference style links](../fig/exercise_links.png)
 >
+>>>>>>> gh-pages
```

Comment:
Looks like this still has the conflict in it...Think this suggestion should fix it?
```suggestion
> to create the following rendered view, where both links point to <https://carpentries.org/>.
>
> ![Preview tab of inline and reference style links](../fig/exercise_links.png)
>
```

Reply:
This one has been updated already!

Reply:
must have been working on a previous commit